lib/tasks/scanning.rake

```diff
@@ -7,4 +7,27 @@ task :brakeman do
   end
 end
 
-task default: "brakeman"
+if Rails.env.development? || Rails.env.test?
```

Comment:
decided just to put all the related rake tasks into our scanning file instead of organizing by namespace in files, felt better

Reply:
I think that's fine (I'd count these as "scanning" anyway). Do we need the rails environment check though?

Reply:
we only install the gems for these files in development and test environments, hence the check, but I think we can remove to make it cleaner and just have any task in this file rightfully break if attempted to run in production